src/server/config.rs

```diff
@@ -61,6 +65,7 @@ impl Default for Config {
             send_buffer_size: DEFAULT_SEND_BUFFER_SIZE,
             recv_buffer_size: DEFAULT_RECV_BUFFER_SIZE,
             end_point_concurrency: DEFAULT_END_POINT_CONCURRENCY,
+            end_point_txn_concurrency_on_busy: DEFAULT_END_POINT_TXN_CONCURRENCY,
```

Comment:
We must check end_point_txn_concurrency_on_busy is greater than zero if DEFAULT_END_POINT_TXN_CONCURRENCY is zero but we forget to call the initialized function outside. 

Or you should use `max(DEFAULT_END_POINT_TXN_CONCURRENCY, 1)` function to guarantee the value.